apps/super-app/src/services/ViewServices.ts

```diff
@@ -1,5 +1,18 @@
 // TODO This is a stub for testing, eventually replace this with Strapi Calls
```

Comment:
> apps/super-app/src/services/ViewServices.ts

Similarly, I'm not sure how to read this file name. Is this a service that serves up `View`s? (I don't know what a View is).